### PR TITLE
WIP browse and search prefs

### PR DESCRIPTION
**Pull request by lmcglohon**

### CODE REVIEW

**Comment by sem42198** [comment]
## Related JIRA Tickets
[ANW-929](https://archivesspace.atlassian.net/browse/ANW-929)
[ANW-685](https://archivesspace.atlassian.net/browse/ANW-685)
[ANW-215](https://archivesspace.atlassian.net/browse/ANW-215)
[ANW-216](https://archivesspace.atlassian.net/browse/ANW-216)
[ANW-681](https://archivesspace.atlassian.net/browse/ANW-681)
[ANW-540](https://archivesspace.atlassian.net/browse/ANW-540)
[ANW-249](https://archivesspace.atlassian.net/browse/ANW-249)

File: frontend/app/models/search.rb [hunk 1]
```diff
@@ -13,7 +13,14 @@ def self.for_type(repo_id, type, criteria)
   def self.all(repo_id, criteria)
     build_filters(criteria)
 
+    puts 'here1'
```
Comment (pobocks) [inline frontend/app/models/search.rb]: This looks like debugging output, should it maybe be dropped or replaced with a log statement, also apply to other `puts` in this method.

Comment (lmcglohon) [reply frontend/app/models/search.rb]: I thought I got rid of those...

File: frontend/app/models/search_result_data.rb [hunk 2]
```diff
@@ -106,15 +106,15 @@ def facet_label_string(facet_group, facet)
     return I18n.t("enumerations.subject_term_type.#{facet.to_s}", :default => facet) if facet_group === "first_term_type"
 
     if facet_group === "source"
-      if single_type? and types[0] === "subject"
+      if get_type and types[0] === "subject"
```
[the remark this answers is not in the document]
Comment (lmcglohon) [reply frontend/app/models/search_result_data.rb]: good catch. Should be able to fix it before the release candidate goes out next week.

[the remark this answers is not in the document]
Comment (lmcglohon) [reply frontend/app/models/search_result_data.rb]: @pobocks I used the second suggestion to parallel what was used for digital_objects. Thanks again for the catch!

**Pull request merged**